cypress.awx.config.ts

```diff
@@ -6,6 +6,7 @@ import { baseConfig } from './cypress.base.config';
 baseConfig.e2e!.specPattern = 'cypress/e2e/awx/**/*.cy.ts';
 baseConfig.e2e!.baseUrl = 'https://localhost:4101';
 baseConfig.component!.specPattern = 'frontend/awx/**/*.cy.{js,jsx,ts,tsx}';
```

Comment:
Should this be reverted?

Reply:
yes, I have been testing the CI flakiness